src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties

```diff
@@ -19,59 +19,38 @@
 
 # default org.apache.maven.plugins groupId
 maven-archetype-plugin=3.2.0
-# https://issues.apache.org/jira/browse/ARCHETYPE-590
```

Comment:
why do you remove this explanation of what was the issue that has been fixed?
I think it's useful for people wanting to understand

Reply:
Issue is addressed and in all information can be retrieved from the SCM log, no?

Reply:
it's really hard to find such info

Reply:
OK, I will drop the hunk and retain the information.